app/src/pages/index.tsx

```diff
@@ -55,9 +55,20 @@ export default function Home({ capabilityStatement }: InferGetServerSidePropsTyp
         <Anchor href="https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html">
           FHIR Measure Repository Service
         </Anchor>{' '}
-        with Measure and Library authoring capabilities.
+        with Measure and Library authoring capabilities. See the{' '}
+        <Anchor href="https://github.com/projecttacoma/measure-repository/blob/main/README.md">
+          Measure Repository README
+        </Anchor>{' '}
+        for technical details.
       </Text>
       <Divider my="sm" variant="dotted" />
+      <Title order={2}>Service Location:</Title>
+      <div style={{ marginTop: '18px', marginBottom: '18px' }}>
+        <Anchor
+          href={`${process.env.NEXT_PUBLIC_MRS_SERVER}/metadata`}
+        >{`${process.env.NEXT_PUBLIC_MRS_SERVER}/metadata`}</Anchor>
```

Comment:
Just checking my understanding on the env variables for the application... will we need to update this to be `process.env.MRS_SERVER` after the docker branch gets merged in order to avoid showing `localhost` as the location when using docker?

Reply:
I believe that's correct.